Asterix/optics/propagation_functions.py

```diff
@@ -328,12 +420,12 @@ def prop_angular_spectrum(pup, lam, z, rad, prad, gamma=2):
 
 
 def fft_choosecenter(image, inverse=False, center_pos='bb', norm='backward'):
-    """
-    FFT Computation. IDL "FFT" routine uses coordinates origin at pixel [0,0].
-             This routine FFTSHIFT2 uses a coordinate origin at any pixel [k,l],
-             thanks to multiplication by adequate array before using numpy routine "FFT".
-             Keywords allow convenient origins either at central pixel ('p') or between
-             the 4 central pixels ('b')
+    """FFT Computation. IDL "FFT" routine uses coordinates origin at pixel.
+
+    [0,0]. This routine FFTSHIFT2 uses a coordinate origin at any pixel [k,l],
```

Comment:
Why is the `[0,0]` here now and what does it mean? It looks like a typo. Also, the docstring looks like a copy-pase from IDL and should be cleaned up. Just replace "This routine FFTSHIFT2" with "This function" and it should be less confusing.

Reply:
ok